### PR TITLE
Bugfix/no ref/hadith scene last part bug fixed

### PR DESCRIPTION
**Pull request by Xayrulloh**

### CODE REVIEW

**Review by eugene0928 — APPROVED** [review]

**Review by eugene0928 — APPROVED** [review]

File: scenes/hadith.ts [hunk 1]
```diff
@@ -36,17 +36,18 @@ scene.wait('hadith').on('message:text', async (ctx) => {
...
+    category: ctx.callbackQuery?.data || ctx.update.message?.text,
   })
 
-  await ctx.editMessageText('The hadith is written, thank you. You are doing your best :)')
+  if (ctx.callbackQuery?.data) {
+    await ctx.answerCallbackQuery()
+
+    await ctx.editMessageText('The hadith is written, thank you. You are doing your best :)')
+  } else {
+    await ctx.reply('The hadith is written, thank you. You are doing your best :)')
```
Comment (toshmukhamedov) [inline scenes/hadith.ts:49]: messageni constant qilib ishlatish kere

**Review by toshmukhamedov — CHANGES_REQUESTED** [review]

File: scenes/hadith.ts (hunk 1 above)
Comment (Xayrulloh) [reply scenes/hadith.ts:49]: Badge keremi halyam))

Comment (toshmukhamedov) [reply scenes/hadith.ts:49]: tushunmadim?

Comment (toshmukhamedov) [reply scenes/hadith.ts:49]: Just bind it to a variable and use it everywhere.

Comment (Xayrulloh) [reply scenes/hadith.ts:49]: Githubdigi badge midi ili bowqa narsami

Comment (Xayrulloh) [reply scenes/hadith.ts:49]: Manga suggestion berses bosvoraman dmoqciman agar kere bosa

Comment (toshmukhamedov) [reply scenes/hadith.ts:49]: erinvosmi dimande 😂

Comment (Xayrulloh) [reply scenes/hadith.ts:49]: Yeah obedlanvomman

File: scenes/hadith.ts [hunk 2]
```diff
@@ -36,17 +36,18 @@ scene.wait('hadith').on('message:text', async (ctx) => {
...
     content: ctx.session.hadith,
-    category,
+    category: ctx.callbackQuery?.data || ctx.update.message?.text,
   })
 
-  await ctx.editMessageText('The hadith is written, thank you. You are doing your best :)')
+  if (ctx.callbackQuery?.data) {
+    await ctx.answerCallbackQuery()
+
+    await ctx.editMessageText('The hadith is written, thank you. You are doing your best :)')
```
Comment (toshmukhamedov) [inline scenes/hadith.ts:47]:
```suggestion
    const message = 'The hadith is written, thank you. You are doing your best :)';
    await ctx.editMessageText(message)
```

File: scenes/hadith.ts (hunk 1 above)
Comment (toshmukhamedov) [inline scenes/hadith.ts:49]:
```suggestion
    await ctx.reply(message)
```

**Review by toshmukhamedov — CHANGES_REQUESTED** [review]

**Review by toshmukhamedov — APPROVED** [review]

**Pull request merged**